guides/hack/15-asynchronous-operations/28-guidelines.md

```diff
@@ -514,7 +515,9 @@ function main(): void {
 }
 ```
 
-This scenario normally occurs in the global scope, but can occur anywhere.
+**THIS IS A COUNTEREXAMPLE**: in real-world code, the entrypoint should be made async instead.
```

Comment:
You removed one reference to "the global scope" (hhvm 4.6x pseudo mains), but this one survived.
`// In a non-async function, or the global scope, you can`